### PR TITLE
[TD] use consistent name for balloon outline

### PR DESCRIPTION
**Pull request by donovaly**
- in the prefs we call the outline "shape" and this describes it better than "symbol" (it is the shape of the bubble). The line end, however, has a symbol so "Shape" makes clear that is is something different.

### CODE REVIEW

File: src/Mod/TechDraw/App/DrawViewBalloon.h [hunk 1]
```diff
@@ -52,8 +52,8 @@ class TechDrawExport DrawViewBalloon : public TechDraw::DrawView
     App::PropertyLink        SourceView;
     App::PropertyString      Text;
     App::PropertyEnumeration EndType;
-    App::PropertyEnumeration Symbol;
-    App::PropertyFloatConstraint SymbolScale;
+    App::PropertyEnumeration Shape;
```
Comment (vocx-fc) [inline src/Mod/TechDraw/App/DrawViewBalloon.h]: If you change the property name, doesn't this break older files that were saved with the previous name?

Comment (donovaly) [reply src/Mod/TechDraw/App/DrawViewBalloon.h]: You are right, stupid me.

Comment (donovaly) [reply src/Mod/TechDraw/App/DrawViewBalloon.h]: This is now fixed: af5007c

**Comment by WandererFan** [comment]
Merged in spite of Travis Linux build error in MeshWB

**Pull request merged**